### PR TITLE
Fix `function-no-unknown` and `custom-property-no-missing-var-function` false positives for `running()`

### PR DESCRIPTION
**Pull request by Mouvedia**
> Which issue, if any, is this issue related to?

#5742

> Is there anything in the PR that needs further explanation?

https://www.w3.org/TR/css-gcpm-3/#running-syntax
https://www.princexml.com/doc/styling/#position
https://www.antenna.co.jp/AHF/help/en/ahf-css6.html#css3-functions

### CODE REVIEW

File: .changeset/silly-brooms-beg.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `function-no-unknown` false positives for `running`
```
Comment (jeddy3) [inline .changeset/silly-brooms-beg.md:5]:
```suggestion
Fixed: `function-no-unknown` false positives for `running()`
```

**Review by jeddy3 — COMMENTED** [review]
Thank you.

A minor suggestion, otherwise LGTM.

File: .changeset/silly-brooms-beg.md (hunk 1 above)
Comment (Mouvedia) [reply .changeset/silly-brooms-beg.md:5]: I was matching the previous changesets.
i.e. 078d7f8, 4d32d3

**Review by jeddy3 — APPROVED** [review]
LGTM, thank you.

> I was matching the previous changesets.

Oh, right. Either would've been fine then.

**Pull request merged**